gallery/psgallery/psgallery_requires_license_acceptance.md

```diff
@@ -0,0 +1,16 @@
+---
+ms.date:  2017-06-12
+contributor:  Farehar
+ms.topic:  conceptual
+keywords:  gallery,powershell,psgallery
+title:  psgallery_require_license_acceptance
+---
+
+Requires License Acceptance
+===========================
+
+Requires License Acceptance text shows up on item details page for modules that require license acceptance. License for module can be viewed by clicking on 'View License.txt' link.
+
+![Require License Acceptance](Images/RequireLicenseAcceptance.png)
+
+Users will be prompted to accept the license when installing, saving or updating the module through PowerShellGet or when deploying to azure automation. To learn more, see [PowerShellGet Documentation](../psget/module/RequireLicenseAcceptance.md)
```

Comment:
Add More details section similar to [https://docs.microsoft.com/en-us/powershell/gallery/psgallery/psgallery_pseditions](https://docs.microsoft.com/en-us/powershell/gallery/psgallery/psgallery_pseditions)

Reply:
Done

---
In reply to: [136225843](https://github.com/PowerShell/PowerShell-Docs/pull/1620#discussion_r136225843) [](ancestors = 136225843)